### PR TITLE
feat: Add IDMS snippets #704

### PR DESCRIPTION
**Pull request by schte01**
Signed-off-by: Teri Schnitt <teri.schnitt@broadcom.com>

Added snippets for shell template, IDMS sections and COPY IDMS statements.

### CODE REVIEW

File: clients/cobol-lsp-vscode-extension/snippets.json [hunk 1]
```diff
@@ -1982,5 +1982,108 @@
 			"$0"
 		],
 		"scope": "cobol"
-	}
+	},
+    "IDMS-CONTROL SECTION ": {
```
Comment (temanbrcom) [inline clients/cobol-lsp-vscode-extension/snippets.json:1986]: I see that some identifiers finish with whitespace before the closing quote. Is it intentional?

File: clients/cobol-lsp-vscode-extension/snippets.json [hunk 2]
```diff
@@ -1982,5 +1982,108 @@
...
+        "prefix": "IDMS-CONTROL-SECTION",
+        "body": [
+            "IDMS-CONTROL SECTION.",
+            "PROTOCOL. MODE IS ${1:operating-mode}."
+        ],
+        "description": "[IDMS] idms-control section",
+        "scope": "cobol"
+    },
+    "SCHEMA SECTION ": {
```
Comment (grianbrcom) [inline clients/cobol-lsp-vscode-extension/snippets.json:1995]: Some names end on spaces. Can you delete those tailing spaces?

Comment (schte01) [reply clients/cobol-lsp-vscode-extension/snippets.json]: I removed the spaces

**Review by temanbrcom — APPROVED** [review]
🦎

**Pull request merged**